src/main/java/frc/robot/subsystems/intake/commands/StartIntake.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.subsystems.intake.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.intake.Intake;
+
+public class StartIntake extends CommandBase {
+    private Intake intake;
+    private boolean direction;
+    public StartIntake(Intake i , boolean direction) {
+        intake = i;

```

Comment:
```suggestion

    public StartIntake(Intake intake, boolean direction) {
        this.intake = intake;

```
and rename the direction boolean, it's unclear what is true and what is false